### PR TITLE
Services, FlowerInfo, SightingInfo, Comments

### PR DESCRIPTION
**Pull request by iObrovac**
@kiljee 
@damjanbanjac

### CODE REVIEW

**Comment by iObrovac** [comment]
@kiljee 
@damjanbanjac

File: src/components/card/Card.tsx [hunk 1]
```diff
@@ -21,27 +20,17 @@ const Card: React.FC<ICardProps> = ({
   const { loggedIn } = useContext<IContext>(UserContext);
   const [flowerInfo, setFlowerInfo] = useState<IExampleFlower>(flowerData);
 
+  const dispatch = useDispatch();
+  const { deleteOneFavFlower } = bindActionCreators(actionCreators, dispatch);
```
Comment (kiljee) [inline src/components/card/Card.tsx:24]: you can import deleteOneFavFlower, 
and try dispatach(deleteOneFavFlower()) don't need bindActionCreators

File: src/components/card/Card.tsx [hunk 2]
```diff
@@ -21,27 +20,17 @@ const Card: React.FC<ICardProps> = ({
...
+  let navigate = useNavigate();
+
```
Comment (kiljee) [inline src/components/card/Card.tsx:26]: const navigate

File: src/components/card/Card.tsx [hunk 3]
```diff
@@ -21,27 +20,17 @@ const Card: React.FC<ICardProps> = ({
...
-        `https://flowrspot-api.herokuapp.com/api/v1/flowers/${flowerData.id}/favorites`,
-        "",
-        {
-          headers: {
-            Authorization: `Bearer ${localStorage.getItem("token")}`,
-          },
-        }
-      );
+      const res = await postLikeFlower(flowerData.id);
+      console.log(res);
```
Comment (kiljee) [inline src/components/card/Card.tsx:30]: delete console.log

File: src/components/card/Card.tsx [hunk 4]
```diff
@@ -21,27 +20,17 @@ const Card: React.FC<ICardProps> = ({
...
       try {
-        const response2 = await axios.get<IGetFavorites>(
-          `https://flowrspot-api.herokuapp.com/api/v1/flowers/favorites?page=1`,
-          {
-            headers: {
-              Authorization: `Bearer ${localStorage.getItem("token")}`,
-            },
-          }
-        );
+        const response2 = await fetchFavorites();
```
Comment (kiljee) [inline src/components/card/Card.tsx:33]: don't use response2

**Pull request merged**